backend/grant/rfp/models.py

```diff
@@ -0,0 +1,61 @@
+import datetime
+from grant.extensions import ma, db
+from grant.utils.enums import RFPStatus
+
+
+rfp_proposal = db.Table(
+    'rfp_proposal', db.Model.metadata,
+    db.Column('rfp_id', db.Integer, db.ForeignKey('rfp.id')),
+    db.Column('proposal_id', db.Integer, db.ForeignKey('proposal.id'), unique=True)
+)
+
+
+class RFP(db.Model):
+    __tablename__ = "rfp"
+
+    id = db.Column(db.Integer(), primary_key=True)
+    date_created = db.Column(db.DateTime)
+
+    title = db.Column(db.String(255), nullable=False)
+    brief = db.Column(db.String(255), nullable=False)
+    content = db.Column(db.Text, nullable=False)
+    category = db.Column(db.String(255), nullable=False)
+    status = db.Column(db.String(255), nullable=False)
+
+    # Relationships
+    proposals = db.relationship("Proposal", secondary=rfp_proposal)
```

Comment:
I see the models are already setup for future many->1 proposal to rfp linking in the UI. Nice.